docker/php/docker-healthcheck.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e
+
+if env -i REQUEST_METHOD=GET SCRIPT_NAME=/ping SCRIPT_FILENAME=/ping cgi-fcgi -bind -connect /var/run/php/php-fpm.sock; then
```

Comment:
```json
    "err_trace": "reverseproxy.statusError (reverseproxy.go:1272)",
    "msg": "dialing backend: dial unix /var/run/php/php-fpm.sock: connect: no such file or directory",
```

Reply:
![image](https://user-images.githubusercontent.com/4578773/210363617-7ae5a5f3-49f1-4513-9c6e-68e8e9aa2730.png)

Reply:
- https://github.com/gammadia/tipee/blob/feature/OPS-unify-php-and-nginx/scripts/docker/s6-rc.d/nginx/run
- https://cloud.google.com/run/docs/configuring/healthchecks